src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -28,7 +28,7 @@
 
 public class IntegrityCheck {
 
-    private final BibDatabaseContext bibDatabaseContext;
+    private static BibDatabaseContext bibDatabaseContext;
```

Comment:
Because I want to use a method of this class in class EditionChecker. An alternative is, to define a `private final BibDatabaseContext bibDatabaseContext;` in class EditionChecker, but this is redundant.